CMakeLists.txt

```diff
@@ -300,6 +293,16 @@ endif()
 #second time
 include_directories(${GISMO_INCLUDE_DIRS})
 
+if(GISMO_WITH_TRILINOS)
+  add_subdirectory(extensions/gsTrilinos)
+  set (GISMO_INCLUDE_DIRS ${GISMO_INCLUDE_DIRS} ${TRILINOS_INCLUDE_DIR}
+  CACHE INTERNAL "${PROJECT_NAME} include directories")
+  #include_directories(${TRILINOS_INCLUDE_DIR})
+endif(GISMO_WITH_TRILINOS)
+
+#third time
+include_directories(${GISMO_INCLUDE_DIRS})
+
```

Comment:
The "second time" could be removed safely probably?

Reply:
No, the "second time" is needed so that Trilinos can find "SuperLU" and "UMFPACK".